system/lib/standalone/standalone.c

```diff
@@ -152,7 +152,7 @@ int emscripten_resize_heap(size_t size) {
 }
 
 double emscripten_get_now(void) {
-  return (1000 * clock()) / (double)CLOCKS_PER_SEC;
+  return (1000ll * clock()) / (double)CLOCKS_PER_SEC;
```

Comment:
Is this related?

Reply:
This was needed to make the benchmarks pass in standalone mode.  Without this we get wrap around in the timings and emscripten_get_now starts returning negative numbers after a while.